charts/ps-operator/README.md

```diff
@@ -0,0 +1,55 @@
+# Percona Operator for Percona Server
+
+Percona Operator for PS allows users to deploy and manage Percona Server Clusters based on async and group replication on Kubernetes.
+Useful links:
+- [Operator Github repository](https://github.com/percona/percona-server-mysql-operator)
+- [Operator Documentation](https://www.percona.com/doc/kubernetes-operator-for-mysql/ps/index.html)
+
+## Pre-requisites
+* Kubernetes 1.20+
+* Helm v3
```

Comment:
https://kubernetes.io/releases/patch-releases/#non-active-branch-history
it looks like 1.20 is not supported anymore. Should we change k8s version too?

Reply:
Since on EKS you can still run 1.19, let's leave it this way for now.